cmd/get.go

```diff
@@ -27,11 +28,21 @@ import (
 )
 
 func get(cmd *cobra.Command, args []string) (err error) {
+	if len(args) < 1 {
+		return errors.New("source is required")
+	}
+
 	src, err := validatePath(args[0])
-	dst := args[1]
+	if err != nil {
+		return
+	}
 
-	if dst == "" {
+	var dst string
+	if len(args) < 2 {
+		// If a destination is not specified use the base of the source path
 		dst = path.Base(src)
```

Comment:
Why not just initialize dst to Base(src) and then use args[1] if len == 2. On that note, should we also error out if len > 2?